rafthttp/peer.go

```diff
@@ -31,6 +31,18 @@ const (
 	ConnWriteTimeout = 5 * time.Second
 
 	recvBufSize = 4096
+
+	streamApp   = "streamMsgApp"
+	streamMsg   = "streamMsg"
+	pipelineMsg = "pipeline"
```

Comment:
we have streamType defined: https://github.com/coreos/etcd/blob/master/rafthttp/stream.go#L37
Any thoughts to integrate them?